flocker/common/test/test_era.py

```diff
@@ -0,0 +1,39 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.common._era``.
+"""
+
+from uuid import UUID
+from unittest import skipUnless
+
+from twisted.trial.unittest import SynchronousTestCase
+from twisted.python.runtime import platform
+from .._era import get_era
+
+
+class EraTests(SynchronousTestCase):
+    """
+    Tests for ``get_era``
+    """
+    @skipUnless(platform.isLinux(), "Only possible on Linux.")
```

Comment:
I'd say "get_era only supported on Linux". This sounds too much like a challenge that will only end with the BSD hordes descending upon you, awash in the glory of the their microkernel.